### PR TITLE
German: More suggested corrections for the German text files

### PR DESCRIPTION
**Pull request by Talandazar**
German: More suggested corrections for the German text files.

### CODE REVIEW

File: crawl-ref/source/dat/descript/de/backgrounds.txt [hunk 1]
```diff
@@ -11,19 +11,19 @@ an Luftmagie-Zaubern die nötige Flexibilität, um zu überleben.
...
-Arkane Scharfschützen sind für den Fernkampf bewaffnet und verwenden Magie, um ihre
+Arkane Schützen sind für den Fernkampf bewaffnet und verwenden Magie, um ihre
 Feinde anfälliger für derartige Angriffe zu machen.
 %%%%
 Artificer
 
-Artificer beginnen mit zwei Zauberstäben, einer magischen Schachfigur und einer
-Keule. Sie müssen bessere Waffen und Rüstungen finden, bevor ihre Zauberstäbe
+Artificer beginnen mit zwei Zauberstäben, einer magischen Schachfigur und einem
+Knüppel. Sie müssen bessere Waffen und Rüstungen finden, bevor ihre Zauberstäbe
```
Comment (powerbf) [inline crawl-ref/source/dat/descript/de/backgrounds.txt:20]: If we change this here, we'll have to also change it in dat/strings/de/item*.txt

File: crawl-ref/source/dat/descript/de/backgrounds.txt [hunk 2]
```diff
@@ -32,16 +32,16 @@ Curare-Wurfpfeile.
...
 %%%%
 Conjurer
 
-Hervorrufer begegnen Probleme mit schädlichen Zaubersprüchen.
+Konjuranten begegnen Problemen mit schädlichen Zaubersprüchen.
 %%%%
 Delver
 
-Gräber beginnen ihre Suche fünf Ebenen unter der Oberfläche. Eine Reihe von
+Entdecker beginnen ihre Suche fünf Ebenen unter der Oberfläche. Eine Reihe von
```
Comment (powerbf) [inline crawl-ref/source/dat/descript/de/backgrounds.txt:44]: We'll also have to change this to match in dat/string/de.

File: crawl-ref/source/dat/descript/de/branches.txt [hunk 3]
```diff
@@ -200,16 +200,16 @@ Ein ausgeklügelter Komplex, der tollkühne Abenteurer auf der Suche nach
 Reichtum und Ruhm anlockt. Wer ihn betritt, muss sich einen Weg durch mehrere
 Kampfarenen bahnen, von denen jede einen Schatz enthält, der von gefährlichen
 und manchmal exotischen Kreaturen bewacht wird. Am Ende wartet ein
-furchterregender Minotaur, der einen großen Haufen Beute von den Leichen
-törichter Abenteurer bewacht.
+furchterregender Minotaur, der einen großen Haufen Beute bewacht, die
+von den Leichen törichter Abenteurer geplündert hat.
```
Comment (powerbf) [inline crawl-ref/source/dat/descript/de/branches.txt:204]: Should it be "die er..."?

**Review by powerbf — CHANGES_REQUESTED** [review]

File: crawl-ref/source/dat/descript/de/ability.txt [hunk 4]
```diff
@@ -18,7 +18,7 @@ Breathe Fire ability
     local desc = "Einen Feuerstrahl auf eine anvisierte Kreatur speien"
 
     if you.race() == "Red Draconian" then
-        desc = desc .. ", und eine Flammenwolke am Endpunkt hinterlassen"
+        desc = desc .. " und eine Flammenwolke am Endpunkt hinterlassen."
```
[the remark this answers is not in the document]
Comment (Talandazar) [reply crawl-ref/source/dat/descript/de/ability.txt:21]: Oh, right. I'll fix it in the next commit and I'll pay more attention to the code from now on.

File: crawl-ref/source/dat/descript/de/branches.txt (hunk 3 above)
Comment (Talandazar) [reply crawl-ref/source/dat/descript/de/branches.txt:204]: Yes, it should! I'll fix it in the next commit.

**Pull request merged**